test/common/schemaform/definitions/phone.unit.spec.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { findDOMNode } from 'react-dom';
+import { expect } from 'chai';
+import ReactTestUtils from 'react-addons-test-utils';
+
+import { DefinitionTester } from '../../../util/schemaform-utils.jsx';
+import { schema, uiSchema } from '../../../../src/js/common/schemaform/definitions/phone';
+
+describe('Schemaform definition phone', () => {
+  it('should render phone', () => {
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          schema={schema}
+          uiSchema={uiSchema()}/>
+    );
+
+    const formDOM = findDOMNode(form);
+
+    const input = formDOM.querySelector('input');
+    expect(input.type).to.equal('tel');
+    expect(input.classList.contains('home-phone')).to.be.true;
+    expect(input.classList.contains('va-input-medium-large')).to.be.true;
```

Comment:
I was doing this because there's a class set in the definition for this field. So I wanted to make sure it was set correctly. Otherwise, yeah, we try to avoid checking for classes too much.